Source/WebKit/_WebKit_SwiftUI/API/WebView.swift

```diff
@@ -152,4 +189,23 @@ extension WebView {
         /// The URL of the link that the user clicked.
         public let linkURL: URL?
     }
+
+    private enum Storage: DynamicProperty {
```

Comment:
oh this should probably go either in an extension of WebView, or at the top of WebView (nested types usually do one of these)

Reply:
I think a `// MARK: Nested Types` could also work. Although I personally think extensions-as-organization are overrated, especially for protocol conformances, since they don’t surface helpful information in Xcode even.